NEWS.md

```diff
@@ -1,4 +1,25 @@
+## 1.3.0 Released
```

Comment:
Shouldn't we increment bugfix version too because there are bug fixes? So this would be 1.3.1 release?

Reply:
If version 1.3.0 was already released, we would have to increment the patch release version for the bug fixes, but as the bug fixes are going into the same release we can keep it at the same minor version